### PR TITLE
Remove python 2 support

### PR DESCRIPTION
**Pull request by niccokunzmann**
This fixes Issue #64, a discussion is still welcome.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -191,6 +192,8 @@ how to go about it.
 Changelog
 ---------
 
+- v1.0.0b
+    - Remove Python 2 support, see `Issue 64 <https://github.com/niccokunzmann/python-recurring-ical-events/issues/64>`_.
```
Comment (Plonq) [inline README.rst:196]: Looks like you're also removing 3.5 and 3.6 support, so that should also be documented.

File: setup.py [hunk 2]
```diff
@@ -127,10 +127,12 @@ def run(self):
         'Intended Audience :: Developers',
         'Natural Language :: English',
         'Operating System :: OS Independent',
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3 :: Only',
```
Comment (Plonq) [inline setup.py:131]: What do these two lines mean? Is `3` just any version of v3, or the latest? And what does the "Only" mean?

File: setup.py [hunk 3]
```diff
@@ -127,10 +127,12 @@ def run(self):
...
         'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
```
Comment (Plonq) [inline setup.py:135]: You've commented 3.10 in travis, so probably should comment it here too so everything matches

File: README.rst (hunk 1 above)
Comment (niccokunzmann) [reply README.rst:196]: Thanks, yes that should be done!

File: setup.py (hunk 3 above)
Comment (niccokunzmann) [reply setup.py:135]: Yes.. Maybe they will integrate 3.10 in the mean time...

**Pull request merged**